akka-http-core/src/test/scala/akka/http/impl/engine/client/LowLevelOutgoingConnectionSpec.scala

```diff
@@ -350,7 +350,7 @@ class LowLevelOutgoingConnectionSpec extends AkkaSpec("akka.loggers = []\n akka.
   class TestSetup {
     val requests = TestPublisher.manualProbe[HttpRequest]
     val responses = TestSubscriber.manualProbe[HttpResponse]
-    val remoteAddress = new InetSocketAddress("example.com", 80)
+    val remoteAddress = new InetSocketAddress("example.com", 0)
```

Comment:
This change seems a bit strange. If the `remoteAddress` is only used to construct the `Host` instance below, replacing the `remoteAddress` with the `Host` object directly seems clearer.